compose/const.py

```diff
@@ -67,4 +69,5 @@
     API_VERSIONS[COMPOSEFILE_V3_4]: '17.06.0',
     API_VERSIONS[COMPOSEFILE_V3_5]: '17.06.0',
     API_VERSIONS[COMPOSEFILE_V3_6]: '18.02.0',
+    API_VERSIONS[COMPOSEFILE_V3_7]: '18.02.0',
```

Comment:
18.06.0?

Reply:
well, I wasn't sure about that. From my understanding, it's the "lowest" API version supported. And so far, there is nothing in v3.7 that wasn't already supported by v3.6 ; this is why I did put the same version.